src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -604,3 +537,32 @@ def get_pg_info(cls):
             'statuses': pg_summary['all'],
             'pgs_per_osd': pgs_per_osd,
         }
+
+
+def _generate_key(encryption_type, backend, config_name):
```

Comment:
let's properly type the params.
```suggestion
def _generate_key(encryption_type: str, backend: str, config_name: str):
```

Reply:
hey Its already there in the code

Reply:
fixed